lib/task-data/base-tasks/generate-tag.js

```diff
@@ -0,0 +1,13 @@
+// Copyright 2016, EMC, Inc.
+
+'use strict';
+
+module.exports = {
+    friendlyName: 'Generate Tag',
+    injectableName: 'Task.Base.Catalog.GenerateTag',
```

Comment:
@yyscamper: The SKU command, which is exactly the same, is called Task.Base.Catalog.GenerateSKU.  I believe they are named this way because they are initiated during discovery cataloging.